runtime/lua/vim/lsp/util.lua

```diff
@@ -303,6 +310,78 @@ function M.make_floating_popup_options(width, height, opts)
   }
 end
 
+-- local function update_tagstack()
```

Comment:
tbh I don't know what this function does, so I'm hiding it but keeping it for reference in case someone wants to do something with it.

Reply:
If it actually works(ed) then following a tag (i.e. "go to definition") will update the list reported by `:tags`, and `ctrl-t` will go back in the "history" of go-to-definition actions.

If it doesn't work, just remove it, but in the `-- update_tagstack` line in `jump_to_location`, mention the SHA so its impl can be found easily.

Reply:
donezo